proc-macro/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use ::quote::{
 #[cfg(not(feature = "test"))]
 macro_rules! CRATE_NAME {() => (
     ::core::convert::identity::<syn::Ident>(syn::parse_str(
-        &::proc_macro_crate::crate_name("function_name")
+        &::proc_macro_crate::crate_name("function_name-proc-macro")
```

Comment:
The `proc_macro_crate::crate_name` lookup is for the _frontend_ crate, so I'd rather keep it as it was, unless you noticed issues with it?

```suggestion
        &::proc_macro_crate::crate_name("function_name")
```

Reply:
I had to change this because tests didn't compile complaining about the reference not found. I actually inserted the name of the inner create.

Reply:
Yeah, the tests actually need to be run with `--features dev` for them to work. I've made a change in #6 that gets rid of that quirk